From f9775b454e1969a8d5c8d8b7435817876ed79482 Mon Sep 17 00:00:00 2001
Message-ID: <f9775b454e1969a8d5c8d8b7435817876ed79482.1692703597.git.pmatilai@redhat.com>
In-Reply-To: <bbb289e303d8c72b9e35410e593b8d92b006bec1.1692703597.git.pmatilai@redhat.com>
References: <bbb289e303d8c72b9e35410e593b8d92b006bec1.1692703597.git.pmatilai@redhat.com>
From: Panu Matilainen <pmatilai@redhat.com>
Date: Mon, 14 Aug 2023 12:31:06 +0300
Subject: [PATCH 2/3] Look for alternative implementations of 7zip, like
 autoconf did

Fixes a regression from the cmake transition where we'd just assume 7zip
as the name of the executable when previously we looked through 7zip,
7za and 7a. Resume the former behavior.

Fixes: #2608
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 30f413028..55c1d2169 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -88,7 +88,7 @@ function(makemacros)
 	set(rundir /run)
 	set(root_prefix /usr)
 
-	findutil(__7ZIP 7zip)
+	findutil(__7ZIP "7zip;7za;7z")
 	findutil(__BZIP2 bzip2)
 	findutil(__CAT cat)
 	findutil(__CHMOD chmod)
-- 
2.41.0

